app/templates/direct-award/index.html

```diff
@@ -52,16 +52,23 @@ <h1>
       empty_message="You have no saved searches",
       field_headings=[
         "Name",
-        "Ended"  ],
+        "Ended",
```

Comment:
Thought occurs that we might need a new word here from @karlchillmaid, perhaps "Finished searching date" or something, if we aren't using the word "end" in relation to searches anywhere now.

Reply:
@karlchillmaid - if you leave your comments here about the table heading, I'll update it

Reply:
Hey @DilwoarH, see below for the changes we discussed earlier that @karlchillmaid and I are suggesting. 

1) "Search ended" table name changes to "Results exported"
2) "Ended" column name changes to "Exported"
3) "Outcome" column name changes to "Status"
4) new state "Download results" for when search has been exported but not yet downloaded (i.e. user shouldn't be able to tell us the outcome just yet). This should link to the *Download your results* page

<img width="1071" alt="screen shot 2018-06-27 at 15 53 01" src="https://user-images.githubusercontent.com/902580/41982483-6c66c7b0-7a23-11e8-88f8-d73d61348a4f.png">

Reply:
Re (4), I think this is very much an edge case, especially if we do combine those steps as originally planned.

However, it would make sense for the "Download results" link to be there right up until they've awarded the contract, regardless of whether they downloaded it already.